Documentation/doc/resources/1.8.14/DoxygenLayout.xml

```diff
@@ -4,7 +4,7 @@
     <tab type="mainpage" visible="yes" title=""/>
     <tab type="modules" visible="yes" title="" intro=""/>
     <tab type="pages" visible="yes" title="" intro=""/>
-    <tab type="classlist" visible="yes" title="Class and Concept List" intro="Here is the list of all concepts and classes of the CGAL Library. Classes are inside the namespace CGAL. Concepts are in the global namespace."/>
+    <tab type="classlist" visible="no" title="Class and Concept List" intro="Here is the list of all concepts and classes of the CGAL Library. Classes are inside the namespace CGAL. Concepts are in the global namespace."/>
```

Comment:
Dummy question: why do we even keep that line? Wouldn't we have the same result if we remove it?

Reply:
I made the minimal change possible, thinking that if one day we want it back it would be easier like this. But you are technically right, I guess. Removing it or hiding it shouldn't be really different in the user's point of view.

Reply:
Thanks.